.github/workflows/deploy.yml

```diff
@@ -36,8 +36,9 @@ jobs:
 
     - name: Deploy to AWS CloudFormation
       run: |
+        sed "s/REPLACE_ME/$(openssl rand -hex 4)/" api/cloudformation_unprocessed.yml > api/cloudformation.yml
         aws cloudformation package \
-          --template-file cloudformation.yml \
+          --template-file cloudformation-replaced.yml \
```

Comment:
Should this have the `-replaced`? It looks like you're `sed` is going from `cloudformation_unprocessed.yml` to `cloudforamtion.yml`

Reply:
yes Thanks